src/views/barang/editdetailbarang.vue

```diff
@@ -16,11 +16,18 @@ export default {
   },
   created() {
     //ambil data by id
-    this.getData()
+    this.getDataDetailBarang()
```

Comment:
`this.getDataDetailBarang(1) // anggap 1 sebagai id nya karena id akan diambil sesuai dengan data yang mau di edit`